### ISSUE TITLE
Krypton compatibility?

### ISSUE THREAD

**Issue by zag2me** [Freso/script.module.requests#12]
Hi just tested the iPlayer Add-on that uses this dependecy and it seems to be failing with Kodi version 17.

Any chance of updating it to work?

10:29:02 T:2104   ERROR: XFILE::CDirectory::Create - Error creating C:\Users\user\AppData\Roaming\Kodi\temp\6b538b0a-a10b-4e99-8f1b-bb4c4fc00fcb
10:29:02 T:2104   ERROR: CAddonInstallJob[plugin.video.iplayerwww]: failed to install dependency script.module.requests

**Comment by beenje** [comment]
I'm away this week, but I'll have a look as soon as I can.

**Comment by beenje** [comment]
I just had a quick look and script.module.requests should still be compatible with Krypton.
I got the same problem as you when trying to install iPlayer on my Mac.
The problem is due to the first error:
10:29:02 T:2104 ERROR: XFILE::CDirectory::Create - Error creating C:\Users\user\AppData\Roaming\Kodi\temp\6b538b0a-a10b-4e99-8f1b-bb4c4fc00fcb

The "temp" directory doesn't exist... I created it manually and the install then worked.

I checked on the bug tracker and this is fixed in a nightly build: http://trac.kodi.tv/ticket/17078
